package.json

```diff
@@ -35,6 +38,7 @@
     "remark": "^15.0.1",
     "remark-heading-id": "^1.0.1",
     "remark-html": "^16.0.1",
+    "sharp": "^0.33.5",
```

Comment:
Hello @MaGOs92 !
Pourquoi a t'on besoin de `sharp` ?

Reply:
Salut Nicolas, c'est une recommandation que j'ai vu dans les logs de build de Scalingo

Reply:
C'est très étrange. Le log precise dans quel cas il a besoin de cette dependances ?
Au pire, est-ce qu'il ne devrait pas être en devDependencies ?

Reply:
Bien vu, je pense que c'était lié au flag NPM_CONFIG_PRODUCTION=false qui était setté dans l'environnement. 
Là j'ai refait un build sans sharp, je ne vois plus de warning

Reply:
En faite c'est pas au build mais au runtime qu'apparait ce log

Reply:
En fait il faut installer la dépendance sharp non présent dans next.js : https://nextjs.org/docs/messages/sharp-missing-in-production